### PR TITLE
Improvement: ollama handle several model files

### PR DESCRIPTION
**Pull request by aboubakar14**

### CODE REVIEW

File: src/configuration/configuration.json [hunk 1]
```diff
@@ -4,10 +4,14 @@
...
     "ai-models":{
         "ollama": {
-            "base_url": "http://localhost:11434/api",
+            "base_url": "http://127.0.0.1:11434/api",
             "models": ["mistral"],
             "defaultModel": "mistral",
-            "modelFiles": ["default-mistral-modelfile"]
+            "modelFiles": {
+                "default": "default-mistral-modelfile",
+                "kubernator-plugin": "default-kubernetes-mistral-modelfile",
```
Comment (Zorin95670) [inline src/configuration/configuration.json]: need to be change:
`kubernator-plugin` -> `@ditrit/kubernator-plugin`
`githubator-plugin` -> `@ditrit/githubator-plugin`

File: tests/unit/configuration/test_configurationManager.py [hunk 2]
```diff
@@ -57,18 +57,29 @@ def test_configuration_validation_error_no_default(
...
+        mocked_config = {
+            "pluginPreferences": {"default": "ollama"},
+            "ai-models": {
+                "ollama": {
+                    "base_url": "http://localhost",
+                    "models": ["mistral"],
+                    "defaultModel": "mistral",
+                    "modelFiles": {
+                        "default": "default-mistral-modelfile",
+                        "kubernator-plugin": "default-kubernetes-mistral-modelfile",
```
Comment (Zorin95670) [inline tests/unit/configuration/test_configurationManager.py]: same here for coherency

File: src/handlers/Ollama/OllamaHandler.py [hunk 3]
```diff
@@ -61,15 +64,16 @@ def __parse_response(self, response_text):
...
             response_text,
             re.DOTALL,
         )
+        print(f"response_text: {response_text}")
         if json_match:
             json_data = json_match.group(1)
             try:
-                json.loads(json_data)
-                return json_data
+                print(f"json_data: {json_data}")
```
Comment (Zorin95670) [inline src/handlers/Ollama/OllamaHandler.py]: why not log ????

File: tests/unit/handlers/Ollama/test_OllamaHandler.py [hunk 4]
```diff
@@ -17,11 +21,15 @@ def setUp(self) -> None:
...
                     "ollama": {
-                        "base_url": "http://localhost:11434/api",
+                        "base_url": "http://localhost",
                         "models": ["mistral"],
                         "defaultModel": "mistral",
-                        "modelFiles": ["default-mistral-modelfile"],
-                    },
+                        "modelFiles": {
+                            "default": "default-mistral-modelfile",
+                            "kubernator-plugin": "default-kubernetes-mistral-modelfile",
```
Comment (Zorin95670) [inline tests/unit/handlers/Ollama/test_OllamaHandler.py]: same here

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

File: src/configuration/configuration.json (hunk 1 above)
Comment (aboubakar14) [reply src/configuration/configuration.json]: done

File: tests/unit/configuration/test_configurationManager.py (hunk 2 above)
Comment (aboubakar14) [reply tests/unit/configuration/test_configurationManager.py]: done

File: src/handlers/Ollama/OllamaHandler.py (hunk 3 above)
Comment (aboubakar14) [reply src/handlers/Ollama/OllamaHandler.py]: done

File: tests/unit/handlers/Ollama/test_OllamaHandler.py (hunk 4 above)
Comment (aboubakar14) [reply tests/unit/handlers/Ollama/test_OllamaHandler.py]: done

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**